sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -282,7 +285,7 @@ public void setParameters(
             resultSet.next();
             int count = resultSet.getInt(1);
 
-            Assert.assertEquals(2000, count);
+            Assert.assertEquals(rowsToAdd + JdbcTestDataSet.EXPECTED_ROW_COUNT, count);
```

Comment:
What would be awesome is if the data set itself were parameterized and had concisely computable verification properties... so that when we dump more data in it doesn't break our ability to run the verification.